src/app/shared/components/footer/footer.component.ts

```
@@ -60,7 +66,28 @@
     this.cameraClicked.emit();
   }
 
+  goToExpenses() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)